libdnf5/module/module_sack.cpp

```diff
@@ -739,6 +739,18 @@ ModuleSack::resolve_active_module_items() {
 }
 
 
+ModuleQuery module_spec_to_query(BaseWeakPtr & base, const std::string & module_spec) {
```

Comment:
I guess that the function should be static or in anonymous name space. Or did I overlooked something?

Reply:
Fixed.